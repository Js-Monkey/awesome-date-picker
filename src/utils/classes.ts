const logo = 'better-datepicker'
export const wrapper = `${logo}-wrapper`
export const header = `${logo}-header`
export const day = `${logo}-day`
export const dayBar = `${day}-bar`
export const dayContent = `${day}-content`
export const month = `${logo}-month`
export const year = `${logo}-year`
export const show = `${logo}-show`
export const hidden = `${logo}-hidden`
export const float = `${logo}-float`
