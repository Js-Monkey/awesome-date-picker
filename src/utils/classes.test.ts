import * as classesName from './classes'

describe('classes', () => {
  test('className', () => {
    expect(classesName).toMatchSnapshot()
  })
})
