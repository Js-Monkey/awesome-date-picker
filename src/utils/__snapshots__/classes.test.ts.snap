// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`classes className 1`] = `
Object {
  "hidden": "fl-datepicker-hidden",
  "show": "fl-datepicker-show",
  "wrapper": "fl-datepicker-wrapper",
}
`;
